source/includes/administration/_pricings.md

```diff
@@ -317,7 +391,35 @@ curl -X POST "https://cloudmc_endpoint/rest/pricings" \
         },
         "cogs": 4,
         "currency": "CAD",
-        "id": "45301951-59cb-461e-b0d0-85c46a7abb69"
+        "id": "45301951-59cb-461e-b0d0-85c46a7abb69",
+        "pricingProducts": [
+		{
+			"product": {
+				"id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3"
+			},
+			"cogs": 4,
+			"unitPrice": 5,
+			"currency": "CAD",
+      "pricingTiers": [
```

Comment:
could you have an example with chunksize? also tiers have to end with an upperBound = null, which is the equivalent to greater than. Is that actually validated in the backend?

Reply:
since without that, there's a gap.

Reply:
4a0987ea4d67135a94dd482003b322dd6017e4fb

Reply:
it's checked in the validateContiguousBounds method in the PricingValidations file

Reply:
you're only checking that the previous upper bounds aren't null, not that the last one has to be null, right? I might be misunderstanding